pypy.py

```
@@ -0,0 +1,93 @@
+print(divide(10, 0))  # Division by zero without any error handling
+
+# Inefficient looping with bad variable naming
+def process_data(data_list):
+    for i in range(len(data_list)):  # Inefficient iteration
+        print("Item: " + data_list[i])  # String concatenation instead of format strings
+
+data = [1, 2, 3, 4, 5]
+process_data(data)
+
+# Use of mutable default argument
+def append_to_list(value, my_list=[]):
```

Ruff (0.14.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)